setup.yml

```diff
@@ -73,3 +73,12 @@
     - matrix-postgres-backup
     - matrix-backup-borg
     - matrix-common-after
+
+
+- name: "Set up additional Jitsi JVB servers"
+  hosts: "jvb_servers"
```

Comment:
Maybe a group named `jitsi_jvb_servers` will be clearer to people reading the `inventory` file in the future

Reply:
Agreed, will change